packages/core/demo/data/gauge.ts

```diff
@@ -0,0 +1,44 @@
+export const gaugeData = [
+	{ group: "value", value: 42 },
+	{ group: "delta", value: -13.37 }
+];
+
+export const gaugeDataNoDelta = [{ group: "value", value: 67 }];
+
+// guage no custom color
+export const gaugeOptionsSemi = {
+	title: "Gauge semicircular -- danger status",
+	resizable: true,
+	height: "250px",
+	width: "100%",
+	gauge: {
+		type: "semi",
+		status: "danger"
+	}
+};
+
+// guage with custom color
+export const gaugeOptionsCircular = {
+	title: "Gauge circular -- warning status",
+	resizable: true,
+	height: "250px",
+	gauge: {
+		status: "warning",
+		type: "full"
+	}
+};
+
+// guage with custom color
+export const gaugeOptionsCircularNoDelta = {
+	title: "Gauge circular without delta",
+	resizable: true,
+	height: "250px",
+	gauge: {
+		type: "full"
+	},
+	color: {
+		scale: {
+			value: "#891EE8"
```

Comment:
what if no color provided? do we default to something?

Reply:
yep! uses model colorscale to default to carbon dataviz palette colors (first 2 demos are using default colors)